ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -108,7 +108,9 @@ public void Send()
 #if WINDOWS_UWP
             CoreDispatcher dispatcher,
 #else
-            Dispatcher notUsed,
+#pragma warning disable CS1573
```

Comment:
The use of "notUsed" produces at least warnings... One for the not used parameter and one for the mismatch in the XML documentation.  Suppressing the not used warning and changing the argument name to match the documentation.